tests/daemon/test-persistent-cache.c

```diff
@@ -1081,7 +1060,7 @@ test_persistent_cache_get_offset_wont_update_timestamps_if_it_isnt_supposed_to (
 
```

Comment:
The documentation for this method has fallen out of sync with the implementation.